repository/repository-web/src/main/resources/static/index.html

```diff
@@ -71,7 +71,7 @@
   <button type="button" class="close" data-dismiss="alert">×</button>
   <span class="glyphicon glyphicon-warning-sign"></span> <strong>Sandbox</strong> The main purpose of this system is to provide an environment for experimenting with Vorto Models and Code Generators. 
   	If you find any issues, please report them <a href="https://github.com/eclipse/vorto/issues">here</a>.
-  	For feedback please contact <a href="mailto:vorto-dev@eclipse.org">vorto-dev@eclipse.org</a>.
+  	For feedback please contact <a href="mailto:vorto-dev@eclipse.org">{{ context.supportEmail }}</a>.
```

Comment:
Hi @erlemantos 
I think u missed out replacing the href using the configured email :)